zhmcclient/_auto_updater.py

```diff
@@ -227,6 +236,58 @@ def __init__(self, updater, session):
         self._session = session
         self._client = None
 
+        # Lazy importing of the stomp module, because the import is slow in some
+        # versions.
+        # pylint: disable=import-outside-toplevel
+        import stomp
+        self._stomp = stomp
+
+        # Indicator whether stomp uses Frame objects for the event listener
```

Comment:
comment+code duplication. could this go into a helper method?

Reply:
done.